### PR TITLE
feat: alias functionality

### PR DESCRIPTION
**Pull request by krackerman**
https://github.com/loopmediagroup/service-search/issues/1910

### CODE REVIEW

File: src/util/rest/alias/get.js [hunk 1]
```diff
@@ -0,0 +1,4 @@
+module.exports = async (call, idx) => call('GET', '', {
+  endpoint: `_cat/aliases/${idx}`
+})
+  .then((result) => (result.body.length === 0 ? null : result.body.map(({ index }) => index)));
```
Comment (krackerman) [inline src/util/rest/alias/get.js:4]: Returning an array right now. Should we just return one index?

File: README.md [hunk 2]
```diff
@@ -371,6 +371,8 @@ Available commands
 Interacting with the rest api of Elasticsearch
 
 - `call(method: String, name: String, options: Object)` - make direct API call to Elasticsearch
+- `alias.get(name: String)` - return the index version for alias
+- `alias.update(name: String)` - update alias for index linking to current index version
```
Comment (simlu) [inline README.md:375]:
```suggestion
- `alias.update(name: String)` - update alias for index, linking to current index version
```

File: src/util/rest/alias/get.js (hunk 1 above)
Comment (simlu) [reply src/util/rest/alias/get.js:4]: I think there should be an assert... and then just return that entry

File: src/util/rest/alias/get.js [hunk 3]
```diff
@@ -0,0 +1,9 @@
+const assert = require('assert');
+
+module.exports = async (call, idx) => {
+  const result = (await call('GET', '', {
+    endpoint: `_cat/aliases/${idx}`
+  })).body;
+  assert([0, 1].includes(result.length));
```
Comment (simlu) [inline src/util/rest/alias/get.js:7]:
```suggestion
  assert([0, 1].includes(result.length), result);
```

**Review by simlu — APPROVED** [review]

**Pull request merged**